Cargo.toml

```diff
@@ -34,7 +34,9 @@ console = "0.15"
 directories = "5"
 dunce = "1"
 flate2 = "1"
-futures-util = { version = "0.3", default-features = false, features = ["sink"] }
+futures-util = { version = "0.3", default-features = false, features = [
```

Comment:
Please keep the one line format, as this allows to easily sort the dependencies alphabetically.